rules/know-the-basic-terms-for-a-video/rule.md

```diff
@@ -0,0 +1,71 @@
+---
+type: rule
+title: Production -  Do you know the basic terms for a video?
+uri: know-the-basic-terms-for-a-video
+authors:
+  - title: Adam Cogan
+    url: https://www.ssw.com.au/people/adam-cogan
+  - title: Andrew Forsyth
+    url: https://ssw.com.au/people/andrew-forsyth
+  - title: Jonty Gardner
+    url: https://www.ssw.com.au/people/jonty-gardner
+  - title: Landon Maxwell
+    url: "https://www.ssw.com.au/people/landon-maxwell "
+  - title: Sofie Hong
+    url: https://www.ssw.com.au/people/sofie-hong
+related:
+  - use-cutaways
+created: 2022-10-31T12:40:29.280Z
+guid: 9a3bc559-98d6-4119-88b5-0b7f4f342f6b
+---
+Understanding basic video terminology allows for better communication in video creation. Using basic video terminology can make our video content more engaging. It's important and useful for anyone involved in video production to know the basic terminology.
+
+## Filming Terminology
+
+### Primary footage (main video)
+
+Primary footage is the footage that "tells" the story. It's the main audio and video that often consists of one or more people discussing a topic or narrative. Primary footage is the predominant format for most documentaries, broadcasts, talk shows, or TV shows.
+
+For example, on the SSW TV YouTube channel, this video mainly consists of the primary footage.
+
+`youtube: https://www.youtube.com/embed/mQyzt4CV_Jw`
+
+**Figure: The [interview footage](https://youtu.be/mQyzt4CV_Jw?t=38) drives the story of the video**
+
+Primary footage can be greatly enhanced by using additional footage known as B-roll.
+
+### B-roll footage
+
+B-roll is used to visually support the primary video in telling the story. For example, if the primary footage talks about the office, the B-roll might be the employees, the team meeting etc.
+
+For example, when the main video/primary footage comes to “our developers are working really hard”, the B-roll footage is the developer at work.
+
+`youtube: https://www.youtube.com/embed/BioFgITYAFY`
+
+**Figure: [B-roll](https://youtu.be/BioFgITYAFY?t=63) of two developers discussing**
```

Comment:
1. Please change from "two" to "2"

As per https://ssw.com.au/rules/use-digits-instead-of-words

Reply:
Hi Tiago,
  1. Please change from "two" to "2"
Done